src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -775,4 +772,19 @@
     <value>Rename "{}"</value>
     <comment>Use '{}' to denote placement for the color scheme name to be renamed</comment>
   </data>
+  <data name="ColorScheme_DeleteButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Delete the current color scheme.</value>
+  </data>
+  <data name="ColorScheme_Rename.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Rename</value>
+  </data>
+  <data name="Profile_DeleteButtonDisclaimerInBox" xml:space="preserve">
+    <value>This profile cannot be deleted because it is shipped in-box.</value>

```

Comment:
@cinnamon-msft to help with wording.

> This profile cannot be deleted because it is part of Terminal's default settings. You can hide it by visiting the Advanced page.

I really do almost think that if we're not letting people delete them we should make the button _effectively delete them_ by hiding.

Reply:
So, I have a few issues with the delete --> hide change:
- we already have a "Hide profile from dropdown" checkbox under the Advanced tab
- deleting and hiding are two completely different operations. Hiding removes it from the dropdown, but you can still select it as the default profile. Deleting completely removes the profile from the system.

I think it'd be better to...
- wait until Application State is implemented (#8324). Particularly the deleting dynamic profile part.
- keep the delete button as disabled for now. But when #8324 comes in, we enable it and update the Application State
- move "Hide" to the general tab and use [this approach](https://github.com/microsoft/terminal/pull/8403#issuecomment-744852588) to denote hidden items

Reply:
I'm afraid of having the same button switch from "hide" and "delete" between different profiles. This may cause confusion and people may accidentally delete a profile they wanted to hide.

I like the idea of moving the "hide profile from dropdown" checkbox above the delete button so at least people can find it more easily.

As for wording, I'm thinking: "This profile cannot be deleted because it is included by default."